common/coin_support/coin_utils.h

```diff
@@ -437,4 +437,19 @@ bool verify_receive_derivation_path(const uint32_t *path, uint8_t depth);
  */
 uint16_t get_account_name(const uint32_t *path, uint16_t account_type, char *account_name, uint8_t out_len);
 
+/**
+ * @brief Generates an user readable derivation path from a uint32_t path array
+ * 
```

Comment:
define behaviour when null is passed. what if output buffer is less than the actual string length?